Anlab.Mvc/Extensions/StringExtensions.cs

```diff
@@ -53,5 +54,22 @@ public static bool IsEmailValid(this string value)
             }
             return Regex.IsMatch(value.ToLower(), emailRegex);
         }
+
+        public static string FormatPhone(this string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                return value;
+            }
+            var phoneNumberUtil = PhoneNumbers.PhoneNumberUtil.GetInstance();
+            var phoneNumber = phoneNumberUtil.ParseAndKeepRawInput(value, "US");

```

Comment:
why are we assuming US?  They get lots of international numbers

Reply:
@srkirkland If I use null, it fails.
And above where I format it, it looks nicer if I use National (US)